pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -687,6 +688,16 @@ func (o *StartBuildOptions) RunStartBuildWebHook() error {
 		body, _ := ioutil.ReadAll(resp.Body)
 		return fmt.Errorf("server rejected our request %d\nremote: %s", resp.StatusCode, string(body))
 	}
+
+	body, _ := ioutil.ReadAll(resp.Body)
+
+	var newBuild buildapi.Build
+	if err = runtime.DecodeInto(kapi.Codecs.UniversalDecoder(), body, &newBuild); err != nil {
```

Comment:
I believe status errors are handled shortly before this, are there some that would be able to make it to this part of the code?